src/commands/deploy/args.js

```diff
@@ -81,6 +81,7 @@ export const latestHelp = () => `
 
 export const latestArgs = {
   '--name': String,
```

Comment:
Currently we don't show this. Because we recommend to use `--project`. 
What should we do? Show both and say `--name` is deprecated?

Reply:
I think the help should not show `--name`, but it should continue working for now. And we should render a deprecation message whenever someone passes the flag.